plugins/woocommerce/tests/e2e-pw/test-data/data.js

```diff
@@ -95,6 +95,24 @@ const storeDetails = {
 			downloadable: 'Downloads',
 		},
 	},
+	liberia: {
+		store: {
+			address: 'addr1',
+			city: 'Kakata',
+			zip: 'Division 1',
+			email: admin.email,
+			country: 'Liberia — Margibi', // corresponding to the text value of the option,
+			countryCode: 'LR',
+		},
+		expectedIndustries: 8, // There are 8 checkboxes on the page (in Liberia), adjust this constant if we change that
```

Comment:
Thanks for looking at this @alopezari ! - you weren't necessarily late, it was more the case that we needed this in sooner rather than later to allow the PRs to continue to flow without issue 😊

I agree `expectedNumberOfIndustries` is a better variable name and I will add a task for this to get it updated (I should be able to update this tomorrow) 👍

Reply:
Awesome, thank you! 🙌